app/konnect/mesh-manager/federate-zone.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Federate a zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+If you already have a zone control plane which is not connected to any global control plane, you can federate it to {{site.konnect_short_name}} in Mesh Manager. [Mesh Manager](/konnect/mesh-manager/) allows you to create, manage, and view your {{site.mesh_product_name}} service meshes using the {{site.konnect_short_name}} platform.
+
+Here are a few benefits of creating a mesh deployment in {{site.konnect_short_name}} instead of using a self-managed setup:
+
+* **Kong-managed global control plane:** By creating your mesh in {{site.konnect_short_name}}, your global control plane is managed by Kong. 
+* **All entities in one place:** You can view all your information, such as entities from {{site.kic_product_name}} (KIC) for Kubernetes, {{site.konnect_short_name}}-managed entities, and now service mesh data, all from one central platform. 
+* **Managed UI wizard setup:** {{site.konnect_short_name}} simplifies the process of creating a mesh by providing a setup wizard in the UI that guides you through the configuration steps.
+
+This guide explains how to federate a zone control plane to {{site.konnect_saas}}. 
```

Comment:
What does federating a zone control plane allow users to do? What types of customer use cases does this solve?